src/App.js

```diff
@@ -1,9 +1,18 @@
+import React from 'react-dom';
 import './App.css';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import SideBar from './components/pages/sideBar';
+import LifeStyle from './components/pages/pages/lifestyle';
 
 function App() {
   return (
-    <div className="App">
-      Hello React App
+    <div>
```

Comment:
In this file we need to put only 2 routes, `AuthorizedRoute` and `UnAuthorizedRoute` so the sidebar should go to `AuthorizedRoute` as found in the file structure